wg-serving/charter.md

```diff
@@ -0,0 +1,83 @@
+# WG Serving Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [wg-governance].
+
+[Kubernetes Charter README]: /committee-steering/governance/README.md
+
+## Scope
+
+Discuss and enhance the support for AI/ML inference workloads in Kubernetes.
+
+
+### In scope
```

Comment:
Contrasting this to https://github.com/SergeyKanzhelev/community-1/blob/135df82737eba050d99268db7ed60b63baa12c9a/wg-batch/charter.md, i think we could tighten the in scope section up.  Maybe 1, 2, 4 could be one bullet, then we have another bullet with a suggestion of the areas that are in scope from the use cases doc?